test/e2e/crds/v1alpha1/backendtrafficpolicy.go

```diff
@@ -125,7 +133,26 @@ spec:
 `
 
 		BeforeEach(func() {
-			s.ApplyDefaultGatewayResource(defaultGatewayProxy, defaultGatewayClass, defaultGateway, defaultHTTPRoute)
+			gatewayName := s.Namespace()
+			By("create GatewayProxy")
+			gatewayProxyName := gatewayName
+			err = s.CreateResourceFromString(fmt.Sprintf(defaultGatewayProxy, gatewayProxyName, s.Deployer.GetAdminEndpoint(), s.AdminKey()))
+			Expect(err).NotTo(HaveOccurred(), "creating GatewayProxy")
+			time.Sleep(time.Second)
+
+			By("create GatewayClass")
+			gatewayClassName := fmt.Sprintf("apisix-%d", time.Now().Unix())
+			err = s.CreateResourceFromString(fmt.Sprintf(defaultGatewayClass, gatewayClassName, s.GetControllerName()))
+			Expect(err).NotTo(HaveOccurred(), "creating GatewayClass")
+			time.Sleep(time.Second)
+
+			By("create Gateway")
+			err = s.CreateResourceFromStringWithNamespace(fmt.Sprintf(defaultGateway, gatewayName, gatewayClassName, gatewayProxyName), s.Namespace())
```

Comment:
CreateResourceFromString uses the namespace of the current test, so there's no need to explicitly specify it for namespace-scoped resources.